src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Properties/AssemblyInfo.cs

```diff
@@ -5,7 +5,3 @@
 using System.Resources;
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
Can remove this. Korebuild feature/msbuild adds this. cref https://github.com/aspnet/KoreBuild/blob/ee0383feffde791108402895fd0c60f5d16f1ad0/build/contentFiles/cs/any/AssemblyInfo.Serviceable.cs